cumulusci.yml

```diff
@@ -146,6 +146,23 @@ flows:
                 options:
                     path: "data/scripts/update_all_records_as_recently_viewed.apex"
 
+    create_users_and_assign_permission_sets:
+        steps:
+            1:
+                task: dx
+                options:
+                    command: "force:user:create --setalias rosa --definitionfile unpackaged/users/rosa.json"
+                ignore_failure: True
+            2:
+                task: assign_permission_sets
+                options:
+                    api_names:
+                        - PMDM_View
+                        - PMDM_Deliver
+                        - PMDM_Manage
+                    user_alias: rosa
+                ignore_failure: True
```

Comment:
This is fine, but n.b. this task will not fail if re-run. It only assigns those Permission Sets not already assigned.